src/test/resources/hbz01.es.json

```diff
@@ -83,6 +79,10 @@
   } ],
   "hbzId" : "HT018454638",
   "id" : "http://lobid.org/resources/HT018454638#!",
+  "inCollection" : [ {
+    "id" : "http://lobid.org/resources/HT014176012#!",
+    "label" : "lobid Ressource"
+  } ],
```

Comment:
Uh - sidecar should(!) work . Just define the URI `http://lobid.org/resources/HT014176012#!` and give it the proper label.